src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngProperty.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    ///     Wrapper represeting an arbitrary property of a CNG key or provider
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]  // The [StructLayout] is here to prevent a spurious ApiReviewer alert. We do not actually depend on the layout of this struct.
```

Comment:
If you haven't already, please track things like this as you build out the library, and then we can do a pass over the contract generator to fix these issues.